explore_weather.py

```diff
@@ -0,0 +1,20 @@
+import pandas as pd
+import matplotlib.pyplot as plt
+import sqlite3 as lite
+
+con = lite.connect('weather.db')
+
+# Create df from database
+df = pd.read_sql_query('SELECT * FROM max_temperature ORDER BY request_time', con, index_col='request_time')
+
+# Plot cities
+df.plot()
+plt.show()
+
+# Check out range across the 30 days for each city
+cities = df.columns
+for city in cities:
```

Comment:
this seems bulky. rename the dataframe for something more descriptive, like `cities_df`. then, simply write `for city in cities_df.columns:`